design.md

```diff
@@ -23,3 +23,15 @@
   
 ### login page
 ![main](imgs/login.png "login")
+
+
+### new event form
+![main](imgs/new_event.png "login")
+[source](https://wireframe.cc/nOFTKC)
```

Comment:
"To" and "From" are dates?  Shouldn't they be in the other order?  Also, calendar pickers, or you just type the date?